microsoft/botframework-solutions
# [Skill Template][TypeScript] Add tests for sub-generator

## Description

- Add tests for the `sub-generator skills`
- Remove `customSkillAdapter` file as a template

Related to #1199 

## Testing Steps

1. Go to `AI\templates\Virtual-Assistant-Template\typescript\generator-botbuilder-assistant`
2. Run `npm install`
3. Run `npm run test`
4. Check that all the tests pass successfully

## Checklist
